docs/API-v2.md

```diff
@@ -12,6 +12,15 @@
 
 vmpooler provides a REST API for VM management.  The following examples use `curl` for communication.
 
+## Major change in V3 versus V2
```

Comment:
This heading is wrong - you have not yet introduced a v3 api.... it kinda sounds like you should though. I will likely revisit this comment. My reading thus far is that you are introducing breaking changes which means you should be implementing them as a v3 api instead of modifying the v2 one. Removing the v2 api in addition to the v1 api is perfectly acceptable if that is the level of change desired, but you should not introduce breaking changes in a versioned api endpoint.

Reply:
Oops I got some wires crossed here when updating the readme (Referring to V2 and V3 of vmpooler gem itself) vs the api doc (Referring to the V2 and V3 versioned endpoints).